server/lib/pbench/test/conftest.py

```diff
@@ -0,0 +1,38 @@
+"""This module builts base setup for Server side testing.
+It is implicitly included by PyTest for all unit tests and provides
+test fixtures and other definitions common across the Server unit tests"""
+import logging
+import pytest
+import tempfile
+
+from pathlib import Path
+
+from pbench import _PbenchLogFormatter, _StyleAdapter
+
+
+@pytest.fixture(scope="session", autouse=True)
+def setup(request, pytestconfig):
+    """Test package setup for pbench-server"""
+
+    # Create a single temporary directory for the "/srv/pbench" and
+    # "/opt/pbench-server" directories.
+    tmp = tempfile.TemporaryDirectory(suffix=".d", prefix="pbench-server-unit-tests.")
+    tmp_d = Path(tmp.name)
+
+    pytestconfig.cache.set("TMP", str(tmp_d))
+    request.addfinalizer(lambda: tmp.cleanup())
+
+
+@pytest.fixture
+def test_logger(pytestconfig):
```

Comment:
Interesting point. Probably not, though I doubt it makes a real difference as I wouldn't expect pytest to be selecting test functions from `conftest.py` ... would it?

Reply:
We'll have to watch this going forward.

Reply:
I would venture to say, "no".  I believe the fact that it is decorated as a fixture will prevent the function from being mistaken (by PyTest) as a _test_, but that doesn't necessarily save the Reader from confusion (and I wouldn't want to depend on this aspect of PyTest...).

Since this PR has already been merged, we should probably open an issue to address this.